packages/web/src/app/AppProviders.tsx

```diff
@@ -25,35 +23,34 @@ type AppContextProps = {
 }
 
 export const AppProviders = ({ children }: AppContextProps) => {
+  const { history } = useHistoryContext()
   return (
-    <Provider store={store}>
-      <PersistGate loading={null} persistor={persistor}>
-        <ConnectedRouter history={history}>
-          <LastLocationProvider>
-            <TrpcProvider>
-              <AudiusQueryProvider>
-                <AppContextProvider>
-                  <ThemeProvider>
-                    <NavProvider>
-                      <ScrollProvider>
-                        <RouterContextProvider>
-                          <MainContentContextProvider>
-                            <HeaderContextProvider>
-                              <ToastContextProvider>
-                                {children}
-                              </ToastContextProvider>
-                            </HeaderContextProvider>
-                          </MainContentContextProvider>
-                        </RouterContextProvider>
-                      </ScrollProvider>
-                    </NavProvider>
-                  </ThemeProvider>
-                </AppContextProvider>
-              </AudiusQueryProvider>
-            </TrpcProvider>
-          </LastLocationProvider>
-        </ConnectedRouter>
-      </PersistGate>
-    </Provider>
+    <ReduxProvider>
```

Comment:
V formation

Reply:
Pretty stoked our redux store initialization is inside the react tree now